hail/python/hail/matrixtable.py

```diff
@@ -2533,6 +2533,63 @@ def _localize_entries(self, entries_field_name, cols_field_name) -> 'Table':
         return Table(CastMatrixToTable(
             self._mir, entries_field_name, cols_field_name))
 
+    @typecheck_method(entries_array_field_name=nullable(str),
+                      columns_array_field_name=nullable(str))
+    def localize_entries(self,
+                         entries_array_field_name=None,
+                         columns_array_field_name=None) -> 'Table':
+        """Represent this matrix as a table of entry-rows.
```

Comment:
This is a very confusing sentence. How about:

"Convert the matrix table to a table with entries localized as an array of structs."